benchmark/scripts/Benchmark_Driver

```diff
@@ -118,9 +118,15 @@ def instrument_test(driver_path, test, num_samples):
     return avg_test_output
 
 
-def get_tests(driver_path):
+
+def get_tests(driver_path, args):
     """Return a list of available performance tests"""
-    return subprocess.check_output([driver_path, '--list']).split()[2:]
+    tests = subprocess.check_output([driver_path, '--list']).split()[2:]
+    if args.filter:
+        return filter(lambda name: name.startswith(args.filter), tests)
+    if not args.benchmark:
+        return tests
+    return sorted(list(set(tests).intersection(set(args.benchmark))))
```

Comment:
This will silently fail if someone passes in a non-matching benchmark. We should either emit a warning or fail in such a case.